### ISSUE TITLE
rp-start is not being called

### ISSUE THREAD

**Issue by eed3si9n** [lightbend/reactive-app-maven-plugin#43]
## steps

```
$ minikube start
$ eval $(minikube docker-env)
$ mvn -DskipTests=true -Dmaven.javadoc.skip=true -Dgpg.skip=true clean install
```

etc and go through the step to install Lagom

## problem

The service is not responsive via ingress.

## expectation

The service works.

## note

@TimMoore and I discovered that the pod is using `deployments/run-java.sh` instead of `rp-start` script, and thus Play inside Lagom was listening on port 9000, instead of `RP_ENDPOINT_0_BIND_PORT` (10000).

**Comment by eed3si9n** [comment]
I wonder if it's because application.conf doesn't have the Play stuff merged in.

```
play.modules.enabled += com.lightbend.rp.test.lagomendpoints.impl.HelloModule
play.crypto.secret = whatever

lagom.services {
  hello-impl = "http://localhost:9000"
}

play.modules.enabled +=
  "com.lightbend.rp.servicediscovery.lagom.javadsl.ServiceLocatorModule"

play.filters.disabled+=play.filters.hosts.AllowedHostsFilter

# TODO: This config should be generated at runtime as it is platform-specific, or this project needs platform-specific
# configs that are included dynamically depending upon platform, perhaps via -Dconfig.resource

akka {
  io {
    dns {
      resolver = async-dns
      async-dns {
        provider-object = "com.lightbend.rp.asyncdns.AsyncDnsProvider"
        resolve-srv = true
        resolv-conf = on
      }
    }
  }
}
akka {
  actor {
    provider = cluster
  }

  cluster {
    shutdown-after-unsuccessful-join-seed-nodes = 40s
  }

  discovery {
    method = reactive-lib

    reactive-lib {
      class = com.lightbend.rp.akkaclusterbootstrap.ClusterServiceDiscovery
    }
  }

  library-extensions += "com.lightbend.rp.akkaclusterbootstrap.ClusterBootstrapAutostart"

  remote {
    netty.tcp {
      hostname = ${?RP_ENDPOINT_AKKA_REMOTE_HOST}
      port = ${?RP_ENDPOINT_AKKA_REMOTE_PORT}

      bind-hostname = ${?RP_ENDPOINT_AKKA_REMOTE_BIND_HOST}
      bind-port = ${?RP_ENDPOINT_AKKA_REMOTE_BIND_PORT}
    }
  }
}

com.lightbend.platform-tooling {
  status {
    health-checks += "com.lightbend.rp.akkaclusterbootstrap.ClusterStatusCheck"
    readiness-checks += "com.lightbend.rp.akkaclusterbootstrap.ClusterStatusCheck"
  }
}

# FIXME: Include these only when in a Lagom project. Mechanism TBD.

lagom {
  cluster {
    exit-jvm-when-system-terminated = on
    join-self = off
  }
}
akka {
  library-extensions += "com.lightbend.rp.akkamanagement.AkkaManagementAutostart"

  management {
    http {
      hostname = ${?RP_ENDPOINT_AKKA_MGMT_HTTP_HOST}
      port = ${?RP_ENDPOINT_AKKA_MGMT_HTTP_PORT}
      bind-hostname = ${?RP_ENDPOINT_AKKA_MGMT_HTTP_BIND_HOST}
      bind-port = ${?RP_ENDPOINT_AKKA_MGMT_HTTP_BIND_PORT}
    }
  }
}
akka {
  library-extensions += "com.lightbend.rp.status.ApplicationStatus"

  management {
    http {
      route-providers += "com.lightbend.rp.status.ApplicationStatus"
    }
  }
}
```

**Comment by TimMoore** [comment]
@eed3si9n was this fixed in #45?

**Comment by TimMoore** [comment]
Yes, fixed in #45
